Makefile

```diff
@@ -146,6 +148,41 @@ install_tfhe_lints:
 	(cd utils/cargo-tfhe-lints-inner && cargo install --path .) && \
 	cd utils/cargo-tfhe-lints && cargo install --path .
 
+.PHONY: create_venv # Create Python virtualenv under venv/
+create_venv:
+	virtualenv venv
```

Comment:
this should be in setup_venv maybe, normally if the venv already exists virtualenv works ok

also I think it would be better to have a call with

python3 -m venv venv

?